app/helpers/application_helper.rb

```diff
@@ -0,0 +1,6 @@
+module ApplicationHelper
+
+  def link_to_image(image_path, target_link,options={})
+  link_to(image_tag(image_path, :border => "0"), target_link, options)
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)